dehelpers/filters/float.py

```diff
@@ -0,0 +1,7 @@
+def like_float(xycell): 
```

Comment:
@osamede20 - I'm super super sorry and ty for changing this but I was wrong, `like_float` isn't quite right either (as it could be an int), but if we use `is_numeric` it'd be perfect (as this is finding ints and floats)

Reply:
@mikeAdamss 
No worries at all Mike. I will effect the changes shortly.
As always, thanks for the feedback.